projects/core/src/occ/adapters/product/default-occ-product-config.ts

```diff
@@ -5,7 +5,7 @@ export const defaultOccProductConfig: OccConfig = {
     occ: {
       endpoints: {
         product:
-          'products/${productCode}?fields=DEFAULT,averageRating,images(FULL),classifications,manufacturer,numberOfReviews,categories(FULL)',
+          'products/${productCode}?fields=FULL,averageRating,images(FULL),classifications,manufacturer,numberOfReviews,categories(FULL)',
```

Comment:
Can we have something more specific than `FULL` here? We are moving away from using `FULL`, 'DEFAULT' and 'BASIC' anyway.